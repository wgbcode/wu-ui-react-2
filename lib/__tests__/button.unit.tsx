import * as renderer from "react-test-renderer";
import React from "react";
import Button from "../button/button.example";

describe("button", () => {
  it("button 测试", () => {
    const json = renderer.create(<Button />).toJSON();
    expect(json).toMatchSnapshot();
  });
});
