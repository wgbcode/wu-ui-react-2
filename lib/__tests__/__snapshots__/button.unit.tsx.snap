// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button button 测试 1`] = `
<button>
  button
</button>
`;
